src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -151,20 +182,37 @@ public void beforeNavigateTo(String url, WebDriver driver) {
                    + "</td><td> <br/> &nbsp;</td></tr>");
     CommonUtils.appendTextToFile(logPath, builder.toString());
     logJSError(driver);
+
+    testLogging.addSteps(new TestStepsLogging("success", "Navigate to", url, true));
   }
 
   @Override
   public void afterNavigateTo(String url, WebDriver driver) {
     StringBuilder builder = new StringBuilder();
+    String status;
     if (!AlertHandler.isAlertPresent(driver)) {
       if (url.equals(driver.getCurrentUrl())) {
         builder.append("<tr class=\"success\"><td>Url after navigation</td><td>"
                        + driver.getCurrentUrl() + "</td><td> <br/> &nbsp;</td></tr>");
         CommonUtils.appendTextToFile(logPath, builder.toString());
+
+        status = "success";
       } else {
+        builder.append("<tr class=\"warning\"><td>Url after navigation</td><td>"
+            + driver.getCurrentUrl() + "</td><td> <br/> &nbsp;</td></tr>");
+        CommonUtils.appendTextToFile(logPath, builder.toString());
+
+        status = "warning";
         logWarning("Url after navigation", driver.getCurrentUrl());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Url after navigation" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)